apps/jira-node/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "helen-integration-jira",
```

Comment:
shouldn't have "helen" in the name of an app, better to call 
"monday-jira-integration"